content/manager/update-deployment.md

```diff
@@ -6,58 +6,70 @@ draft: false
-##### via an archived blueprint
+#### via an archived blueprint
 ```shell
 cfy deployments update -d ID_OF_DEPLOYMENT_TO_UPDATE -l ARCHIVE_PATH [-n BLUEPRINT_FILENAME]
 ```
-When updating a deployment using an archive, the name of the blueprint representing the deployment update is assumed to be `blueprint.yaml`. If the blueprint file has a different name, it must be specified using the `-n / --blueprint-filename` optional argument.
+When updating a deployment using an archive, the name of the blueprint representing the deployment update is assumed to be `blueprint.yaml`. If the blueprint file has a different name, it must be specified using the `-n / --blueprint-filename` argument.
 
-{{% gsInfo title="Some Deployment Update Concepts" %}}
-* The **deployment update blueprint** is the blueprint that contains the changes representing the deployment update, what was previously referenced as the 'updated application blueprint'.
-* **step** is a logical concept that represents a single change in a deployment update blueprint. There are three different step types: **add**, **remove**, and **modify**. These three concepts will be used extensively throughout this guide. The scope of a 'step' is determined by the most top-level change. For example, if a node was added, and this node also contains relationship, then this is a 'add node' step, and not a 'add relationship' step. Currently, After you apply a deployment update, its composing step are only accessible using the Cloudify REST API.
+{{% gsInfo title="Some deployment update concepts" %}}
+- The **deployment update blueprint** is the blueprint that contains the changes representing the deployment update, what was previously referenced as the 'updated application blueprint'.
+- A **step** is a logical concept that represents a single change in a deployment update blueprint. There are three different types of steps: **add**, **remove**, and **modify**. These three concepts will be used extensively throughout this guide. The scope of a 'step' is determined by its most top-level change. For example, if a node was added, and this node also contains a new relationship, then this is a 'add node' step, and not a 'add relationship' step. Similarly, if a node's property was modified, it is not a 'modify node' step, but a 'modify property' step. A list of all the possible steps is located [here](http://docs.getcloudify.org/3.4.0/manager/update-deployment/#what-can-be-updated-as-a-part-of-a-deployment-update)
 {{% /gsInfo %}}
+{{% gsNote title="Accessing the steps of a deployment update" %}}
+Currently, After you apply a deployment update, its composing steps are only accessible using the Cloudify REST API.
+{{% /gsNote %}}
 
 ### Deployment Update Flow
 Updating a deployment consists of several stages:
 1. The deployment update blueprint is uploaded to the manager.
 2. The steps composing the deployment update are extracted.
 3. All the 'added' changes are updated in the data model.
```

Comment:
Namedropping of 'data model'. This term is not familiar to all. maybe add a link to an explanation, or don't mention.
Do most users even care about 'behind the scenes'? Maybe just add stuff like that as an advanced side note.